src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/permissions/PermissionsClient.scala

```diff
@@ -126,6 +126,20 @@ trait PermissionsClient {
 
   }
 
+  def listOwners(agoraEntity: AgoraEntity): Seq[String] = {
+    // you're a manager if your role & (1 << 4) != 0 (and I think we're in the range from 0-32) so basically 16-31 inclusive
+    val permissionsQuery = for {
+      entity <- entities if entity.alias === alias(agoraEntity)
+      _permissions <- permissions if _permissions.entityID === entity.id && (_permissions.roles inSetBind List.range(16, 32))
+      user <- users if user.id === _permissions.userID
+    } yield user.email
```

Comment:
Instead of hardcoding numbers here, please use the variables in `AgoraPermissions`. Perhaps you can just check for `>= AgoraPermissions.Manage`? There's no need right now to check less-than 32, is there? That would only be relevant if we added a new permission (and that permission was less powerful than `Manage`.

I know I pointed you at `inSetBind` earlier, and maybe that led you astray - I didn't know how the permissions were actually stored, sorry!